lnrpc/rpc.proto

```diff
@@ -1766,8 +1788,19 @@ message QueryRoutesRequest {
     /// The 33-byte hex-encoded public key for the payment destination
     string pub_key = 1;
 
-    /// The amount to send expressed in satoshis
-    int64 amt = 2;
+    /**
+    The amount to send expressed in satoshis.
+
+    The fields amt_sat and amt_msat are mutually exclusive.
+    */
+    int64 amt_sat = 2;
```

Comment:
Any reason to not deprecate `amt_sat`?

Reply:
I think it was brought up that msat causes confusion. I'd rather deprecated sat completely though.

Reply:
@Roasbeef did you have an opinion about the necessity of sat fields?

Reply:
Discussed offine. No deprecation yet.